scripts/ci/jobs/push-images.sh

```diff
@@ -44,63 +44,26 @@ push_images() {
         push_context="merge-to-master"
     fi
 
-    # TODO(ROX-11889): make this unconditional once the openshift/release side is ready
-    if [[ -n "${OPERATOR_IMAGE:-}" ]]; then
-      push_operator_image_set "$push_context" "$brand"
+    if is_OPENSHIFT_CI && is_in_PR_context && pr_has_label "turbo-build" && [[ "$brand" == "RHACS_BRANDING" ]]; then
+        info "Images were built and pushed elsewhere, skipping it here."
+    else
+        push_main_image_set "$push_context" "$brand"
+        push_matching_collector_scanner_images "$brand"
     fi
-    push_main_image_set "$push_context" "$brand"
-    push_matching_collector_scanner_images "$brand"
     if [[ -n "${PIPELINE_DOCS_IMAGE:-}" ]]; then
         push_docs_image
     fi
     if [[ -n "${MAIN_RCD_IMAGE:-}" ]]; then
         push_race_condition_debug_image
     fi
-
-    if is_in_PR_context && [[ "$brand" == "STACKROX_BRANDING" ]]; then
-        comment_on_pr
+    # TODO(ROX-11889): make this unconditional once the openshift/release side is ready
+    if [[ -n "${OPERATOR_IMAGE:-}" ]]; then
+        push_operator_image_set "$push_context" "$brand"
```

Comment:
Note that this either has to happen before the main images, or the polling in the operator e2e job needs to be adjusted to look for the operator image instead of main.
Otherwise, the operator e2e job will time out before the operator bundle index image materializes on quay.

Reply:
Thanks for spotting this. I will look at the operator build (how it was done in CCI) and see if I can add it to this PR. If it looks like too much of a tangent I will change the operator-e2e tests to poll for the operator image.

Reply:
I've added the operator build and https://issues.redhat.com/browse/ROX-12054 to track improvements to the poller.